contracts/Paper.sol

```diff
@@ -51,13 +58,43 @@ contract Paper {
         require(block.timestamp <= deadlineDate, "The deadline date has passed.");
         _;
     }
+
+    modifier primaryCheckedHasPassed() {
+        require(primaryChecked == true);
+        _;
+    }
+
+    function primaryChecking() public onlyOwner() {
+        primaryChecked = true;
+    }
     
-    function addFeedback(string memory _feedback) public canBeReview() {
-        feedbacks.push(FeedBack(_feedback, msg.sender));
+    function addFeedback(string memory _feedback) public  primaryCheckedHasPassed() canBeReview() {
```

Comment:
Add a comment or create a new type alias to show that this string should be an IPFS hash.